ecs-init/docker/docker.go

```diff
@@ -453,17 +455,26 @@ func getCapabilityExecBinds() []string {
 
 	var binds []string
 
-	// bind mount the entire /host/dependency/path/exec/bin folder for higher flexibility
-	// minimal change required to add other ssm binaries as dependency in the future (just need to be placed inside the bin directory)
+	// bind mount the entire /host/dependency/path/execute-command/bin folder
 	hostBinDir := filepath.Join(hostResourcesDir, capabilityExecHostBinRelativePath)
 	if isPathValid(hostBinDir, true) {
-		binds = append(binds, hostBinDir+":"+filepath.Join(containerResourcesDir, capabilityExecContainerBinRelativePath)+readOnly)
+		binds = append(binds,
+			hostBinDir+":"+filepath.Join(containerResourcesDir, capabilityExecContainerBinRelativePath)+readOnly)
 	}
 
-	// bind mount this specific cert file for now, CertsDir and CertsFile might be changed to be configurable in the future
+	// bind mount the entire /host/dependency/path/execute-command/config folder
+	// in read-write mode to allow ecs-agent to write config files to host file system
+	hostConfigDir := filepath.Join(hostResourcesDir, capabilityExecHostConfigRelativePath)
```

Comment:
why is config folder needed for capability detection? Initially, we wont have any config in the AMI, so this check feels unnecessary

Reply:
it needs to be bind mounted into the acs agent container in read/write mode

Reply:
right, we want to make sure that there is 1 place for the agent to write files to